src/SFA.DAS.AssessorService.Web/StartupConfiguration/AuthenticationStartup.cs

```diff
@@ -246,10 +146,10 @@ public static class AuthenticationStartup
                                      identity.AddClaim(new Claim(
                                         "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn",
                                         user?.Username));
-                                     if (user.EndPointAssessorOrganisationId != null)
+                                     if (user.OrganisationId != null)
                                      {
                                          var organisation =
-                                            await orgClient.GetEpaOrganisation(user.EndPointAssessorOrganisationId);
+                                            await orgClient.GetEpaOrganisationById(user.OrganisationId?.ToString());
```

Comment:
As mentioned in Slack, we need to sort out this whole EpaOrganisation Organisation stuff.